dir.props

```diff
@@ -8,6 +8,12 @@
     <OsEnvironment Condition="'$(OsEnvironment)'==''">$(OS)</OsEnvironment>
   </PropertyGroup>
 
+  <!-- Version numbers for both managed & native binaries -->
+  <PropertyGroup>
```

Comment:
can I commit this and open another issue for it. I believe this does not need to be blocked.